protoc-gen-openapiv2/internal/genopenapi/template.go

```diff
@@ -117,31 +117,52 @@ func listEnumNames(reg *descriptor.Registry, enum *descriptor.Enum) (names []str
 		}
 		names = append(names, value.GetName())
 	}
-	return names
+
+	if len(names) > 0 {
+		return names
+	}
+
+	return nil
 }
 
-func listEnumNumbers(reg *descriptor.Registry, enum *descriptor.Enum) (numbers []string) {
+func listEnumNumbers(reg *descriptor.Registry, enum *descriptor.Enum) (numbers []int) {
 	for _, value := range enum.GetValue() {
 		if reg.GetOmitEnumDefaultValue() && value.GetNumber() == 0 {
 			continue
 		}
 		if !isVisible(getEnumValueVisibilityOption(value), reg) {
 			continue
 		}
-		numbers = append(numbers, strconv.Itoa(int(value.GetNumber())))
+		numbers = append(numbers, int(value.GetNumber()))
 	}
-	return
+
+	if len(numbers) > 0 {
+		return numbers
+	}
+
+	return nil
```

Comment:
Please use the same format as the other function
```suggestion
	return numbers
```